packages/transition-backend/src/services/executableJob/ExecutableJob.ts

```diff
@@ -180,12 +180,19 @@ export class ExecutableJob<TData extends JobDataType> extends Job<TData> {
             progressEmitter !== undefined
                 ? progressEmitter
                 : clientEventManager.getUserEventEmitter(this.attributes.user_id);
-        await this.save();
-        execJob('task', [this.attributes.id], {
-            on: function (payload) {
-                jobProgressEmitter.emit(payload.event, payload.data);
-            }
-        });
+        //TODO TEMPORARY SKIP ENQUEUE CHILD JOB (ONES WITH DEFINED PARENT ID)
+        //WE HAVE NOT YET IMPLEMENT A SYNCHRONISATION/NOTIFICATION MECANISM BETWEEN CHILD AND PARENT
+        //SO FOR NOW THE PARENT WOULD NEED TO RUN THE JOB MANUALLY
+        if (this.attributes.parentJobId == null) {
```

Comment:
===, not ==

Reply:
but I want to cover the undefined case (I guess) so I should add || ... === undefined then ?